com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/DebugUtility.java

```diff
@@ -312,6 +316,34 @@ public static void resumeThread(ThreadReference thread) {
         }
     }
 
+    /**
```

Comment:
Better to put another class ThreadUtility. There will be StackFrameUtility definitely. If all utility in DebugUtility, this file will be bloated.

Reply:
There are many places to reference to these methods. It would cause many changed files in this PR. Prefer to use a separate PR to do this.